packages/at_onboarding_cli/example/README.md

```diff
@@ -1,29 +1,40 @@
 List of steps to run the examples for checking apkam enrollment
 
-1. Onboard an atsign which has privilege to approve/deny enrollments
-   dart example/onboard.dart <atsign> <path_store_keys_file>
-   e.g. dart example/onboard.dart @alice🛠 /home/alice/.atsign/@alice🛠_wavikey.atKeys
-2. Authenticate using the onboarded atsign
-   dart example/apkam_authenticate.dart <atsign> <path_of_keys_file_from_#1>
-   e.g. dart example/apkam_authenticate.dart @alice🛠 /home/alice/.atsign/@alice🛠_wavikey.atKeys
-3. Run client to approve enrollments
-   dart example/enroll_app_listen.dart <atsign> <path_of_keys_file_from_#1>
-   e.g dart example/enroll_app_listen.dart @alice🛠 /home/alice/.atsign/@alice🛠_wavikey.atKeys
+1. Onboard an atsign which has privilege to approve/deny enrollments:<br>
+   - run: `dart example/onboard.dart -a <atsign> -c <CRAM secret> -k <path_store_keys_file>`<br>
+   - e.g. `dart example/onboard.dart -a @alice -k /home/alice/.atsign/@alice_wavikey.atKeys`<br/>
```

Comment:
e.g should include -c <cram_secret>

Reply:
change made as part of [fc488f0](https://github.com/atsign-foundation/at_libraries/pull/442/commits/fc488f0742ad967a406ba94aea50a2dc7cfbb1b6)